Formula/db-vcs.rb

```diff
@@ -0,0 +1,16 @@
+class DbVcs < Formula
+  desc "Version control for MySQL databases"
+  homepage "https://github.com/infostreams/db"
+  url "https://github.com/infostreams/db/archive/1.0.tar.gz"
+  sha256 "a21f717ead07058242f28d90bd3d56f478f05039f0628e8f177c4383c36efefd"
+
+  def install
+    libexec.install "db"
+    libexec.install "bin/"
+    bin.install_symlink libexec/"db"
+  end
+
+  test do
+    assert_equal "fatal: Not a db repository (or any of the parent directories). Please run 'db init'.", shell_output("#{bin}/db server add localhost", 2).strip
```

Comment:
Would it be possible to check `db init .` as well?

Reply:
Not really, because `db init` will start an interactive questionnaire (it waits for user input via the keyboard, multiple times). I'm not sure what will happen if there's no TTY – basically that's undefined behavior at this point.